dockerfiles/testsuite/env.miner0

```diff
@@ -9,4 +9,4 @@ VOCDONI_VOCHAIN_GENESIS=/app/misc/genesis.json
 VOCDONI_VOCHAIN_MINERKEY=cda909c34901c137e12bb7d0afbcb9d1c8abc66f03862a42344b1f509d1ae4ce
 VOCDONI_METRICS_ENABLED=True
 VOCDONI_METRICS_REFRESHINTERVAL=5
-VOCDONI_VOCHAIN_MINERTARGETBLOCKTIMESECONDS=5
+VOCDONI_VOCHAIN_MINERTARGETBLOCKTIMESECONDS=6
```

Comment:
Why this change ? Is it because you want to keep the refresh interval lower that the target block time ?

Reply:
Because 6 is the reality. I have not been able to achieve 5s